### ISSUE TITLE
[kickasstorrents-to] (testing) Test search in kickasstorrents.to => Found no results while trying to browse this tracker. This may be an issue with the indexer, or other indexer settings such as search freeleech only etc.

### ISSUE THREAD

**Issue by p060477** [Jackett/Jackett#14961]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
An error occurred while testing this indexer
Test search in kickasstorrents.to => Found no results while trying to browse this tracker. This may be an issue with the indexer, or other indexer settings such as search freeleech only etc.
```

### Description

 An error occurred while testing this indexer
Test search in kickasstorrents.to => Found no results while trying to browse this tracker. This may be an issue with the indexer, or other indexer settings such as search freeleech only etc.

### Logged Error Messages

```text
An error occurred while testing this indexer
Test search in kickasstorrents.to => Found no results while trying to browse this tracker. This may be an issue with the indexer, or other indexer settings such as search freeleech only etc.
```

**Comment by ilike2burnthing** [comment]
All 3 indexers are working fine here.

Provide the environment information as initially requested, rather than just copying and pasting the error 3 times.

**Comment by ilike2burnthing** [comment]
1. please stop declaring everything a 'BUG'
2. there are three separate areas when creating an issue because you are supposed to provide three difference pieces of information, all explained clearly
3. do not open multiple issues about the same problem for different indexers
4. do not open new issues when you're yet to provide basic information for old ones

**Comment by p060477** [comment]
here it is all you need....
https://github.com/FlareSolverr/FlareSolverr/issues/1034#issue-2070796844

**Comment by ilike2burnthing** [comment]
That's a completely different issues (being unable to connect to FlareSolverr when using a VPN). 

Again, provide the requested environment information:

> Please provide the details of the system Jackett is running on. The best way to do this is to restart Jackett, access the dashboard, view the log, and copy/paste the last 10 Info lines at the bottom of the log.

**Comment by p060477** [comment]
01-08 23:43:10 Info Starting Jackett v0.21.1519
01-08 23:43:13 Info Environment version: 6.0.25 (G:\back up 29.04.2010\C\Documents and Settings\Administrator\Dati applicazioni\JACKETT\Jackett 0.18.875\)
01-08 23:43:13 Info OS version: Microsoft Windows NT 10.0.19045.0 (64bit OS)
01-08 23:43:13 Info Jackett variant: CoreWindows
01-08 23:43:13 Info Running in Docker: No
01-08 23:43:13 Info ThreadPool MaxThreads: 1023 workerThreads, 1000 completionPortThreads
01-08 23:43:13 Info App config/log directory: C:\ProgramData\Jackett
01-08 23:43:13 Info Using proxy: Disabled
01-08 23:43:13 Info Using FlareSolverr: http://192.168.2.3:8191
01-08 23:43:13 Info Using HTTP Client: HttpWebClient2
01-08 23:43:13 Info Loading Native indexers ...
01-08 23:43:13 Info Loaded 73 Native indexers.
01-08 23:43:14 Info Loading Cardigann indexers from: C:\ProgramData\Jackett\cardigann\definitions\, G:\back up 29.04.2010\C\Documents and Settings\Administrator\Dati applicazioni\JACKETT\Jackett 0.18.875\Definitions

**Comment by p060477** [comment]
i'm using jackett in a -portable way-...not installed on the system....i really do want to use much more portable sw as i can...in order to be able to use them where ever i want...and to have a very full and usefull back ujp if needed...so i really do want to continue to use jackett in -portabnle- way as up to now....it works so fine ..apart these 8 indexers...the other 106 are working fine...so you have to support also a non portable way...not only the installed ones...

**Comment by ilike2burnthing** [comment]
Open cmd, run the following:

```cmd
dir "G:\back up 29.04.2010\C\Documents and Settings\Administrator\Dati applicazioni\JACKETT\Jackett 0.18.875\Definitions" > G:\result.txt
```

Upload the `result.txt` file in the root of the G drive here.

**Comment by ilike2burnthing** [comment]
Close Jackett if running, rename `C:\ProgramData\Jackett\Indexers` to `C:\ProgramData\Jackett\Indexers.old`, start Jackett again, add all the previously problematic indexers and test.

**Comment by ilike2burnthing** [comment]
Your actual problem (indexer json files unable to update, likely due to permissions issues) has been resolved, so the indexers are now using the correct domains, rather than years old ones or redirects. As for the indexers you're now having problems with:

YourBittorrent, TorrentFunk, and Torlock are all currently down.

Torrentz2eu was down, seems to be back up now, test again.

ProPorno test will fail as there are no results with 2024 in the title, so if you can confirm that searching for `2023` returns no results then you can provide an **enhanced** log or the `2023` search over at https://github.com/Jackett/Jackett/issues/14971 (not here!).

Postman and Cpasbien are being dealt with elsewhere.

I've found GloDLS and Kickasstorrents.to (and LimeTorrents) to be temperamental when using a proxy/VPN, so try multiple servers multiple times (or preferably without, if you're able to access them).

Which just leaves torrent.by and BTSOW. If the errors for them are different than the one here (found no results), then open new issues for them, providing the information requested. If the error is that no results were found, then provide and enhanced log for a test for both indexers - https://github.com/Jackett/Jackett/wiki/Troubleshooting#how-to-provide-a-jackett-enhanced-log-for-a-ticket

**Comment by p060477** [comment]
An error occurred while testing this indexer
Exception (torrentz2eu): The proxy tunnel request to proxy 'http://127.0.0.1:4444/' failed with status code '500'.": The proxy tunnel request to proxy 'http://127.0.0.1:4444/' failed with status code '500'."

**Comment by ilike2burnthing** [comment]
Disable the I2P proxy in Jackett's settings and try again.

**Comment by ilike2burnthing** [comment]
As for ProPorno, perform a search for `2023`, didn't think I needed to explain that part...

**Comment by p060477** [comment]
> As for ProPorno, perform a search for `2023`, didn't think I needed to explain that part...

here it is:

![2023 search](https://github.com/Jackett/Jackett/assets/12773559/07660521-1c49-4a62-9698-86c14f5e6594)

**Comment by ilike2burnthing** [comment]
jfc... in Jackett!

**Comment by p060477** [comment]
> Disable the I2P proxy in Jackett's settings and try again.

done...:

 An error occurred while testing this indexer
Exception (torrentz2eu): Operazione di I/O terminata a causa dell'uscita dal thread oppure della richiesta di un'applicazione.: The request was canceled due to the configured HttpClient.Timeout of 100 seconds elapsing.

**Comment by p060477** [comment]
> jfc... in Jackett!

i do not understand..apologize...what have i to do...??...explain as you were talk to a very little and unclever child...thxs

**Comment by ilike2burnthing** [comment]
torrentz2eu is being very slow and may timeout occasionally, try again later.

For BTSOW - https://github.com/Jackett/Jackett/wiki/TroubleShooting#a-connection-attempt-failed

For ProPorno, click the green search button (magnifying glass icon) beside the indexer and search `2023`. That's probably the most basic function of Jackett.

**Comment by p060477** [comment]
2023 search...in **Jackett**...:

![2023 search](https://github.com/Jackett/Jackett/assets/12773559/2738f23a-7c04-493e-bc2d-6b6ed5332fe5)

**Comment by ilike2burnthing** [comment]
K, that's working fine for you then.

**Comment by p060477** [comment]
> K, that's working fine for you then.

...what you mean...??...pls explain me as i were i little and unclever child...thxs...

**Comment by ilike2burnthing** [comment]
It. Is. Working.

And the test will work tomorrow - https://github.com/Jackett/Jackett/commit/272fc5b2b5e2c7e103e8e062e99d5d041a3e474f

**Comment by p060477** [comment]
so now the errors lefts are 6:

YourBittorrent  : https://yourbittorrent.com/
TorrentFunk: https://www.torrentfunk.com/
Torlock : https://www.torlock2.com/
ProPorno: https://proporno.org/
Postman: http://tracker2.postman.i2p/
Cpasbien: https://cpasbien-vf.fr/

**Comment by ilike2burnthing** [comment]
As I already explained (several times over), the first 3 are down, proporno is working fine for you and the test will work tomorrow, postman is also working fine you just need to retry/wait, and for cpasbien your IP and VPN IP are being blocked.

We're done, and after all that there were zero 'bugs'.

**Comment by p060477** [comment]
> As I already explained (several times over), the first 3 are down, proporno is working fine for you and the test will work tomorrow, postman is also working fine you just need to retry/wait, and for cpasbien your IP and VPN IP are being blocked.
> 
> We're done, and after all that there were zero 'bugs'.

for cpsabien my ip vpn ip is impossible to be blocked...cause i easly reach the web page in browser...
as  i attached to you several times..:
![cpasbien](https://github.com/Jackett/Jackett/assets/12773559/3de37a0c-c609-4704-b1cd-c9c569fdf112)

so this BUG still remains....

**Comment by p060477** [comment]
cpasbien your IP and VPN IP are being blocked : FALSE

....how if i easly reach the webpage...as i posted here with several screenshoots...??

**Comment by p060477** [comment]
hi now the only 3 non working indexers are:
YourBittorrent  : https://yourbittorrent.com/
TorrentFunk: https://www.torrentfunk.com/
Torlock : https://www.torlock2.com/

cause, as you have already stated, they are down
so my question is: why are in jackett if they are down...??...for what is really worth...??
only for loosing time in tryin the tests in vain...??
my question is if they are down and non working why are still suggested to be added in jackett as public indexers...??

**Comment by garfield69** [comment]
We give all web sites up to 30 days to resolve their issues before we remove the corresponding jackett indexer.
This is because the issue with a web site may be temporary, for a variety of reasons, such as:
the domain expired and is waiting on renewal
the  CA is expired and needs renewal
the server needs updating / upgrading / is being moved to a new provider
the web page has an error and needs to be corrected
the database is broken and need validating / fixing
storage is full needs more storage
etc. etc. etc.

**Comment by p060477** [comment]
hi, first thxs for yr clear reply
and then after 30 days if nothing change...??...when i will start my jackett concole and i will open the gui in http://127.0.0.1:9117/UI/Dashboard
and what we will find...??...the 3 will be removed automatically...??...or i will have to do it by myself...??
or it will be done with a yr new upgraded vers of jackett...??
thxs so much in adv for yr kind helping
cheers

**Comment by ilike2burnthing** [comment]
> We give all web sites up to 30 days to resolve their issues **before we remove the corresponding jackett indexer**.

They are removed on your end when you update Jackett.

**Comment by p060477** [comment]
you mean when you will issue a new vers. ,like for ex the last v0.21.1536, ?
thxs in adv
cheers

**Comment by ilike2burnthing** [comment]
Yes.

**Comment by p060477** [comment]
why if i do a search in qbittorent using jackett plug in i got errors in indexers that instead passed the test if i do it directly in jackett gui..??
ex:
Jackett: connection error for indexer: limetorrentsclone! Right-click this row and select 'Open description page' to open help. Configuration file: 'C:\Users\Luca\AppData\Local\qBittorrent\nova3\engines\jackett.json' Search: 'perfect days 2023'
Jackett: connection error for indexer: torrentscsv! Right-click this row and select 'Open description page' to open help. Configuration file: 'C:\Users\Luca\AppData\Local\qBittorrent\nova3\engines\jackett.json' Search: 'perfect days 2023'
Jackett: connection error for indexer: badasstorrents! Right-click this row and select 'Open description page' to open help. Configuration file: 'C:\Users\Luca\AppData\Local\qBittorrent\nova3\engines\jackett.json' Search: 'perfect days 2023'
Jackett: connection error for indexer: itorrent! Right-click this row and select 'Open description page' to open help. Configuration file: 'C:\Users\Luca\AppData\Local\qBittorrent\nova3\engines\jackett.json' Search: 'perfect days 2023'
Jackett: connection error for indexer: aniRena! Right-click this row and select 'Open description page' to open help. Configuration file: 'C:\Users\Luca\AppData\Local\qBittorrent\nova3\engines\jackett.json' Search: 'perfect days 2023'
Jackett: connection error for indexer: idope! Right-click this row and select 'Open description page' to open help. Configuration file: 'C:\Users\Luca\AppData\Local\qBittorrent\nova3\engines\jackett.json' Search: 'perfect days 2023'
Jackett: connection error for indexer: extratorrent-st! Right-click this row and select 'Open description page' to open help. Configuration file: 'C:\Users\Luca\AppData\Local\qBittorrent\nova3\engines\jackett.json' Search: 'perfect days 2023'

these are 7 errors for 7 indexers that in the gui jackett dashboard passed the test and are ok....

...why...??..

**Comment by ilike2burnthing** [comment]
Follow the instructions given to you there.

**Comment by p060477** [comment]
obviously already done before writing here...
you are only redirect here:
https://github.com/qbittorrent/search-plugins/wiki/How-to-configure-Jackett-plugin
where there is really NO answer to the specific problem...it is only a very generic wiki page for generic problems...
but as you can easly see by yourself therer is really no answer to what i have simply ask here:
" 7 errors for 7 indexers that in the gui jackett dashboard passed the test and are ok...."
thxs in adv
cheers

**Comment by ilike2burnthing** [comment]
So you've edited the config file as per the clear instructions in that link which are specific to your issue?

**Comment by p060477** [comment]
...but have you read the link...??....
https://github.com/qbittorrent/search-plugins/wiki/How-to-configure-Jackett-plugin
where are the " clear instructions in that link which are specific to the issue"....??..
it is a generic link and it is the same whatever indexer error you get....
to be clear is the same page that you get:
 if you try to use Postman without i2p proxy setted...or cpsabien without having a french ip....
and where are the info for solving those 2 issues...??
...only opening an issue here we have been able finally, with the consequently yr issuing of the latest vers of jackett ....

**Comment by ilike2burnthing** [comment]
Have you edited the configuration file? If not, follow the instructions under the `Configuration file` section.

**Comment by p060477** [comment]
...but obviously i did...otherwise i should not be able to do a qbittorrent search using only the jackett plugin...
but have you seen the image i've attached...??...:
https://private-user-images.githubusercontent.com/12773559/295733693-d3761381-aca4-434c-b796-d5a06396a4fd.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ5Mjk1MDIsIm5iZiI6MTcwNDkyOTIwMiwicGF0aCI6Ii8xMjc3MzU1OS8yOTU3MzM2OTMtZDM3NjEzODEtYWNhNC00MzRjLWI3OTYtZDVhMDYzOTZhNGZkLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTEwVDIzMjY0MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJlZWM4MmIxYjA0YjJlMDdlNWRjNTYyMzc5ZTJhNDYxY2YyMDViZTQ0Zjc2YTZlN2UwOWRiNDc0MTEzM2FmYzgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.l3NH2a-d4CG4bwNczaMU0IiaDsr8XQEcyypw-GfIpPo

anyway this is my jackett.jason config file :

{
    "api_key": "**here i've put the api key of jackett gui dashboard**",
    "thread_count": 20,
    "tracker_first": false,
    "url": "http://127.0.0.1:9117"
}

**Comment by ilike2burnthing** [comment]
1. there's nothing 'obvious' about what you might or might not have done unless you actually tell me
2. stop posting pointless screenshots, they're yet to have been relevant or helpful
3. stop reposting logs, that does nothing but clutter the issue
4. lose the snark, it's getting old

The answers for your question lies in Jackett's logs (no, don't post it), but at a guess:
- torrents-csv.ml and itorrent.ws are down (torrents-csv.ml fakes the test in Jackett)
- AniRena and Postman timed out (might work now, might not)
- iDope, ExtraTorrent.st, and Badass Torrents required FlareSolverr, which either wasn't running or took too long
- some random issue with LimeTorrents clone

Look at the logs, solve the non-issues yourself based on them, there's no problem here other than the waste of yet more of my time.

**Comment by ilike2burnthing** [comment]
Again, the screenshots don't help (I understood the issue, you just refused to answer a basic question so we could proceed). If we need them we'll ask for them.

Posting logs is fine, reposting them multiple times is a waste of space. Similarly, spamming the same comment on multiple issues is unhelpful.

From all of your issues and comments, the main problem found was your setup. Beyond that, one indexer was updated to change where it runs its test/keywordless search (or we could have just waited for some results with `2024` in the title), another indexer had its TLD updated from .`ml` to `com`, and another enabled empty search inputs. None of those are 'bugs' and you didn't contribute anything to them!

That's the end of this conversation.
